src/code_1AF230.c

```diff
@@ -1,3 +1,21 @@
 #include "common.h"
 
-INCLUDE_ASM(s32, "code_1AF230", LoadStarPowerScript);
+typedef struct StarPower { void* dmaStart; void* dmaEnd; void* dmaDest; Bytecode* init; } StarPower;
```

Comment:
Could you write these in the same style we do other structs (fields on their own line, comments for offsets)?

Reply:
Also this should be named something like "StarPowerEntry"